config/locales/en/lookup.yml

```diff
@@ -10,8 +10,10 @@ en:
         input_error: Enter a valid postcode
         error_description: Enter a valid postcode
     lookup:
-      input_label: Enter your postcode
-      hint_text: For example LS1 1UR
+      input_label: Enter a full postcode 
+      hint_text: |
+        We use this to give you more accurate information about the area.<br/>
+        For example, LS1 1UR
```

Comment:
```suggestion
        <div class="govuk-!-margin-bottom-2">We use this to give you more accurate information about the area.</div>
        <div>For example, LS1 1UR</div>
```

Reply:
To be fair, that's harder than using the hint class.